src/ngraph/frontend/onnx_import/core/value_info.hpp

```diff
@@ -68,7 +73,7 @@ namespace ngraph
             ValueInfo& operator=(ValueInfo&&) = delete;
 
             const std::string& get_name() const { return m_value_info_proto->name(); }
-            const Shape& get_shape() const { return m_shape; }
```

Comment:
[NON-BLOCK]
`get_shape()` can be helpful in some places.
It can get a shape from `m_partial_shape`.(or throw an exception if it isn't static)